presto-main/src/main/java/com/facebook/presto/sql/planner/ExpressionSymbolInliner.java

```diff
@@ -51,8 +61,18 @@ public Expression rewriteSymbolReference(SymbolReference node, Void context, Exp
         @Override
         public Expression rewriteLambdaExpression(LambdaExpression node, Void context, ExpressionTreeRewriter<Void> treeRewriter)
         {
-            // Lambda does not support capture yet. As a result, relation/columns can not exist in lambda.
-            return node;
+            List<String> addedNames = new ArrayList<>();
+            for (LambdaArgumentDeclaration argument : node.getArguments()) {
+                String argumentName = argument.getName();
+                // We must be careful when removing the exclusion because the same argument name may be excluded multiple times.
```

Comment:
I understand this is equivalent to `// This code must handle recursive calls` comment elsewhere?